xml/admin_monitoring_alerting.xml

```diff
@@ -366,36 +366,102 @@ placement:
-    &cephadm;. To do so, you need to enable it in the Pillar under the
-    <option>monitoring:alertmanager_receiver_snmp:enabled</option> key. The
-    configuration of the receiver must be set under the
-    <option>monitoring:alertmanager_receiver_snmp:config</option> key.
+    &cephadm;. To do so, you need to create a service and placement
+    specification file with the following content:
    </para>
-   <example>
-    <title>SNMP Trap Configuration</title>
+   <note>
+     <para>
+       For more information on service and placement files, see
+       <xref linkend="cephadm-service-and-placement-specs"/>.
+     </para>
+   </note>
+<screen>
+service_type: container
+service_id: prometheus-webhook-snmp
+placement:
+    <replaceable>ADD_PLACEMENT_HERE</replaceable>
+image: registry.suse.com/ses/7/prometheus-webhook-snmp:latest
+args:
+    - "--publish 9099:9099"
+envs:
+    - RUN_ARGS="--metrics"
+EOF
+</screen>
+     <para>
+      This service specification is used to get the service running using
+      its default settings.
+     </para>
+     <para>
+      You need to publish the port the &prometheus; receiver is listening on by
+      using the command line argument <literal>--publish <replaceable>HOST_PORT</replaceable>:<replaceable>CONTAINER_PORT</replaceable></literal>
+      when running the service because the port is not exposed automatically by the
+      container. This can be done by adding the following lines specification:
+     </para>
+<screen>
+args:
+    - "--publish 9099:9099"
+</screen>
+     <para>
+       Alternatively, connect the container to the host network by using
+       the command line argument <literal>--network=host</literal>.
+     </para>
 <screen>
-monitoring:
- alertmanager:
-   receiver:
-      snmp:
-        enabled: True
-        config:
-          host: localhost
-          port: 9099
-          snmp_host: snmp.foo-bar.com
-          snmp_community: private
-          metrics: True
+args:
+    - "--network=host"
 </screen>
      <para>
-      Refer to the receiver manual at
-      <link xlink:href="https://github.com/SUSE/prometheus-webhook-snmp#global-configuration-file" />.
-      for more details about the configuration options.
+      Additionally, the SNMP host needs to be configured when the
+      container is connection to the host network via <literal>--network=host</literal>.
+      Use the container's network gateway to be able to receive SNMP traps
+      outside the container:
      </para>
-    </example>
-  </sect1>-->
+<screen>
+envs:
+    - ARGS="--debug --snmp-host=<replaceable>CONTAINER_GATEWAY</replaceable>"
+</screen>
+     <sect2 xml:id="configure-prometheus-">
+       <title>Configure the prometheus-webhook-snmp service</title>
+       <para>
+         The container can be configured by environment variables or by
+         using a configuration file.
+       </para>
+       <para>
+         Use <literal>ARGS</literal> to set global options and <literal>RUN_ARGS</literal>
```

Comment:
Can we have links to the documentation of those environment variables in the origin project? 

ARGS=https://github.com/SUSE/prometheus-webhook-snmp#global
RUN_ARGS=https://github.com/SUSE/prometheus-webhook-snmp#command-run